src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/IdentifiersShouldNotContainUnderscores.Fixer.cs

```diff
@@ -19,11 +25,33 @@ public sealed override FixAllProvider GetFixAllProvider()
             return WellKnownFixAllProviders.BatchFixer;
```

Comment:
We should not support a FixAll here unless we are sure we can offer a good replacement for all violations.

Reply:
@mavasani So, what should I return here?

Reply:
I think your approach in https://github.com/dotnet/roslyn-analyzers/pull/3706#discussion_r434098613 might be good enough to leave this as-is and enable a FixAll to make similar replacements everywhere.